organisation/domain1/team1/rfc000.md

```diff
@@ -0,0 +1,21 @@
+---
+author: Alvaro Alonso
+date: 2023-12-15
+visibility: public
+status: draft
```

Comment:
**thought**: After a few conversations on the PR view this gets moved to a considering or agreed status